kmath-stat/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ kotlin.sourceSets {
         languageSettings.apply {
             useExperimentalAnnotation("kotlinx.coroutines.FlowPreview")
             useExperimentalAnnotation("kotlinx.coroutines.ExperimentalCoroutinesApi")
+            useExperimentalAnnotation("kotlinx.coroutines.ObsoleteCoroutinesApi")
```

Comment:
Do we need that?

Reply:
This line removes compiler warning. It could have been placed to gradle-tools project, actually.